eru/src/components/modules/buttons/regularButton.vue

```diff
@@ -5,7 +5,7 @@
     class="font-bold dark:bg-blue-light dark:hover:bg-blue-accent bg-blue-light hover:bg-blue-accent text-sky-white rounded-curl px-4 py-1 h-9"
     :class="classes"
     :id="id"
-    @click="clickEvent()"
+    @click.prevent="clickEvent()"
```

Comment:
why have you prevent here? @Micheduc25

Reply:
because when we click on the save button in the profile form, it reloads the page automatically. So i had to add the prevent modifier to prevent that default behavior of the button in forms.

Reply:
> 

ok, that is because you have use the <form> tag

Reply:
but it's ok

Reply:
> > 
> 
> ok, that is because you have use the "form" tag

Yes the person who created the page used the form tag. but I think there is no problem with that